experimental/origin-dapp2/src/pages/App.js

```diff
@@ -50,11 +52,18 @@ class App extends Component {
         </div>
       )
     }
+
+    const shouldRenderNavbar = () => {
```

Comment:
can this just be a variable instead of a function? like `const shouldRenderNavbar = this.props.location.pathname !== '/welcome'`

Reply:
Uff I over complicated this one... thanks!